registration-system/admin/pages_mail.php

```diff
@@ -138,7 +141,7 @@ private function transformContacts() {
     }
 
     private function buildQueryWhere() {
-        $where = ['fahrt_id' => $this->fahrt->getID(), 'OR' => ['on_waitlist' => 0,
+        $where = ['fahrt_id' => $this->fahrt->getID(), 'OR #waitlist' => ['on_waitlist' => 0,
```

Comment:
was ist das denn mit der raute? neue medoo Syntax?
Sieht so aus, als sollte da `on_waitlist` stehen (?)

Reply:
das mit komentar muss so. es gibt ja nun auch ein OR weiter unten bei nights. zwei or MUSS man mit medoo so lösen leider. wegen unique map keys

Reply:
ersetzt durch 'getPossibleDates' in der neuen version

Reply:
kannte ich noch nicht, dachte das wäre nur irgendwie die neue syntax oder so.